frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions/ExpressionEditorSuggestions.tsx

```diff
@@ -251,8 +259,9 @@ function ExpressionEditorSuggestionsListItem({
   }, [index, onHighlightSuggestion]);
 
   return (
-    <HoverParent>
+    <HoverParent as="li">
       <ExpressionListItem
+        as="div"
```

Comment:
I checked, `ExpressionListItem` is used only here, which means that instead of using `as="div"`, we could just change the styled element into `styled.div`

Reply:
We're in the process of getting rid of styled components. We're using Mantine whenever possible.

But furthermore, `ExpressionListItem` is also used for the `GroupTitle` which should then also need to be updated.
```
export const GroupTitle = styled(ExpressionListItem)`
  font-weight: bold;
  font-size: 12px;
  color: var(--mb-color-text-medium);
  pointer-events: none;
  margin-top: 12px;

  &:first-child {
    margin-top: 0;
  }
`;
```

Reply:
`Box` or `Flex` are the most commonly used ones.
Flex seems to be the better choice here.

Reply:
I couldn't find anything better than this in just a few minutes when going through some of my closed PRs.
https://github.com/metabase/metabase/pull/40136/files

Hope this helps.